direct/nn/build_nn_model.py

```diff
@@ -0,0 +1,66 @@
+# coding=utf-8
+# Copyright (c) DIRECT Contributors
+
+from enum import Enum
+
+from torch import nn
+
+from direct.nn.conv.conv import Conv2d
+from direct.nn.didn.didn import DIDN
+from direct.nn.resnet.resnet import ResNet
+from direct.nn.unet.unet_2d import NormUnetModel2d, UnetModel2d
+
+
+class ModelName(str, Enum):
+    unet = "unet"
+    normunet = "normunet"
+    resnet = "resnet"
+    didn = "didn"
+    conv = "conv"
+
+
+def _build_model(
```

Comment:
It feels that this method more like returns some configurations for different models? Rather than actually building anything.
I also think it would be a bit cleaner to just send in the model class, but depending on the context in which this is used it might not be.
i.e. something like this:
`kwargs = _get_model_config(UnetModel2d)`